templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/common/impl/config/ApplicationObjectMapperFactory.java

```diff
@@ -29,5 +35,7 @@ public ApplicationObjectMapperFactory() {
 	// register spring-data Pageable
     module.addSerializer(Pageable.class, new PageableJsonSerializer());
     module.addDeserializer(Pageable.class, new PageableJsonDeserializer());
+    // configure properties for Java Time module
+    setConfigPropertiesJavaTime(this.configProps);
```

Comment:
Please remove and instead override `createInstance()` method using `super.createInstance()` and tweak the result with the 3 suggested options. This way we do not change the behaviour of existing projects (potentially breaking anything) and only offer these configs as suggested for new projects.